PORTS.md

```diff
@@ -0,0 +1,37 @@
+# Ports Documentation
+
+Eclipse Thingweb uses a cloud instance provided by Eclipse Foundation to host its services on [plugfest.thingweb.io](http://plugfest.thingweb.io/). 
+In order to function as a TCP/UDP server, we have some ports open to the public Internet.
+These are documented below, alongside the service using the port.
+
+## TCP Ports
+
+| TCP PORT | Service |
+|----------|---------|
+|   80     |  Apache Web Server  |
+|   8080   |  Available       |
```

Comment:
Another generic comment... instead of "Available" I would rather like to see "-" or so which makes it much easier to see which ports are available..

Reply:
Done :)